code/obj/railing.dm

```diff
@@ -0,0 +1,344 @@
+// this is such a dumb joke
+/obj/meme
+	name = "glorious"
+	desc = "FUCK WHAT THE JESUS IS THAT"
+	icon = 'icons/obj/160x160.dmi'
+	icon_state = "dumb-joke-left"
+	layer = 100
+	plane = 100
```

Comment:
Hardcoded plane numbers are not great and might prevent this from displaying at all in the future. However, I admit that we don't have a plane defined for what you're doing here so probably leave it as is and I'll change it in the future when I do some more plane refactoring.

Reply:
Also this is even above HUD, I assume that's intended?

Reply:
> 
> 
> Also this is even above HUD, I assume that's intended?

Yes, lol. At first it wasn't intended, as I only intended it to appear above the title screen. However the result of seeing them overlap the HUD ingame was entertaining enough that I kept it! They're only intended to be used in the title anyway.